cogs/welcome/welcome.py

```diff
@@ -97,14 +99,45 @@ async def sendWelcomeMessageChannel(self, newUser: discord.Member, guild: discor
             message += splitMessage[x]
             if x < len(splitMessage) - 1:
                 message += newUser.mention
-        fullMessage = f"""{message} Welcome to SFU Anime Club's Discord!
 
-Please check out <#225044755887685632> for all of our basic information and rules, and feel free to introduce yourself with the pinned template in this channel!
-
-We hope you enjoy your stay~
+        joinLogEnabled = await self.config.guild(newUser.guild).logJoinEnabled()
+        joinLogChannel = await self.config.guild(newUser.guild).logJoinChannel()
+        errorChannel = discord.utils.get(
+            newUser.guild.text_channels, id=joinLogChannel
+        )
```

Comment:
I think we can remove this, my comment was only for the logger parts

Reply:
Done